clients/v1_repl/v1_repl.go

```diff
@@ -0,0 +1,921 @@
+                   .s'                                                          :y.''.y
+                    -s                                                           mo:::'
+                     ::                                                          yh
+                      //                                      ''''               /M'
+                       o+                                    .s///:/.            'N:
+                        :+                                   /:    -s'            ho
+                         's-                               -/s/:+/.+h'            +h
+                           ys'                            ':'    '-.              -d
+                            oh                                                    .h
+                             /o                                                   .s
+                              s.                                                  .h
+                              -y                                                  .d
+                               m/                                                 -h
+                               +d                                                 /o
+                               'N-                                                y:
+                                h:                                                m.
+                                s-                                               -d
+                                o-                                               s+
+                                +-                                              'm'
+                                s/                                              oo--.
+                                y-                                             /s  ':+'
+                                s'                                           'od--' .d:
+                                -+                                         ':o: ':+-/+
+                                 y-                                      .:+-      '
+                                //o-                                 '.:+/.
+                                .-:+/'                           ''-/+/.
+                                    ./:'                    ''.:o+/-'
+                                      .+o:/:/+-'      ''.-+ooo/-'
+                                         o:   -h///++////-.
+                                        /:   .o/
+                                       //+  'y
+                                       ./sooy.`)
+}
+
+// The logic for the main prompt that is run in the REPL loop
+func (c *Client) executor(cmd string) {
+	if cmd == "" {
+		return
+	}
+	defer c.writeCommandToHistory(cmd)
+	cmdArgs := strings.Split(cmd, " ")
+	switch strings.ToLower(cmdArgs[0]) {
+	case "quit", "exit":
+		color.HiBlack("Goodbye!")
+		os.Exit(0)
+	case "gopher":
+		c.gopher()
+
+	// TODO: Allow user to change API in REPL session
```

Comment:
Is there an issue for this TODO? We should link one here.

Reply:
Now linked here: https://github.com/influxdata/influx-cli/issues/389